tests/main.cpp

```diff
@@ -11,6 +11,25 @@
 
 #include <thread>
 
+template<class Function>
+class TestThread : public arangodb::Thread {
+public:
+
+  TestThread(Function&& f, int i, char** c) :
+    arangodb::Thread("catch"), f_(f), i_(i), c_(c) {
+    run();
+  }
+  void run() {
+    result_ = f_(i_,c_);
+  }
+  int result() {return result_;}
+private:
+  Function f_;
```

Comment:
i thought our convention for private variables is:
`_private`
not:
`private_`

please fix

Reply:
Done